src/app/components/dropdown/index.html

```diff
@@ -1,15 +1,176 @@
-<sky-demo-page
-  pageTitle="Dropdown"
-  skyuxModule="SkyDropdownModule"
-  packageName="@skyux/popovers"
+<sky-tabbed-demo-page
+  pageTitle="Dropodown"
 >
 
-  <sky-demo-page-summary>
-    <p>
-      The dropdown component renders a button that displays a menu when users click it. The menu displays a set of actions or functions. <strong>Do not use the dropdown component as a select input</strong> for users to set the menu's value to one of the options. If you need a select input, see the <a routerLink="/design/guidelines/select-items">options to select items from lists</a>.
-    </p>
-  </sky-demo-page-summary>
+<sky-demo-page-summary>
+  <p>
+    The dropdown component renders a button that displays a menu when users click it. The menu displays a set of actions or functions.
+  </p>
+</sky-demo-page-summary>  
 
+<sky-demo-page-example>
+  <sky-dropdown-demo>
+  </sky-dropdown-demo>
+</sky-demo-page-example>  
+  
+<sky-tabset id="content-tabs">
+
+  <sky-tab
+    tabHeading="Design">
+    
+    <sky-demo-page-content sectionHeading="Usage">
+      <h3 class="sky-text-success">Use when</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Use dropdowns when users require access to multiple, related actions or when users require access to multiple, different actions and there is not enough space to use separate buttons for each action.</p>
```

Comment:
Maybe this is nitpicky, but I think "different" is the wrong word here. It doesn't really convey the difference we are suggesting. I think we use "distinct" or "unrelated" or something like that instead. I'm going to change this to "unrelated," but let me know if that doesn't work or if anyone prefers a different option.

Reply:
Nailed it! I think "unrelated" is much clearer

Reply:
Awesome! ... Also, at the end of this sentence, `multiple, unrelated actions and there is not enough space to use separate buttons for each action` keeps nagging at me. It just reads awkwardly, and I think it makes the whole sentence more difficult to digest. What do you think about changing it to `multiple, unrelated actions  and you do not have room for multiple buttons`? I feel like that tightens up a bit and makes it feel less like a run-on sentence. What do you think?

Reply:
I like condensing it, but not sure about using the second person and addressing the reader.
How about "... and there is not enough room for multiple buttons"?

Reply:
It's hard to put my finger on, but my best guess is that I'm reacting to the implied specificity of **you**. It's a tiny thing (and probably irrelevant), but when guidelines are universal, they don't just apply when **you** are in a set of circumstances - they apply any time those circumstances are true.
Whatever, this is probably pointless hair-splitting. I won't go to war over it, and we can follow our own conventions.